webapp/shop/cred/views.py

```diff
@@ -293,76 +300,71 @@ def cred_exam(trueability_api, **_):
 
 @shop_decorator(area="cred", permission="user", response="html")
 @canonical_staff()
-def cred_provision(trueability_api, **_):
+def cred_provision(ua_contracts_api, trueability_api, **_):
+    contract_item_id = flask.request.args.get("contractItemID")
```

Comment:
If you do `contract_item_id = flask.request.args.get("contractItemID", type=int)` you can remove the cast on line `321`

Reply:
Done :+1: